spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/utils/RewriteRowLevelOperationHelper.scala

```diff
@@ -87,7 +87,8 @@ trait RewriteRowLevelOperationHelper extends PredicateHelper with Logging {
 
   protected val FILE_NAME_COL = "_file"
   protected val ROW_POS_COL = "_pos"
-  protected val AFFECTED_FILES_ACC_NAME = "affectedFiles"
+  // `internal.metrics` prefix ensures the accumulator state is not tracked by Spark UI
+  protected val AFFECTED_FILES_ACC_NAME = "internal.metrics.merge.affectedFiles"
```

Comment:
I guess this one is important.

Reply:
@aokolnychyi I didn't know about this. Thanks Anton !!